web/cypress/integration/Home/EditTest.spec.ts

```diff
@@ -7,19 +7,8 @@ describe('Edit Test', () => {
   it('should edit a test', () => {
     cy.location('pathname').then(pathname => {
       const testId = getTestId(pathname);
-      cy.visit(`/`);
       cy.wait('@testList');
       cy.editTestByTestId(testId);
     });
   });
-
-  it('should edit a test from the test details', () => {
-    cy.location('pathname').then(pathname => {
-      const testId = getTestId(pathname);
```

Comment:
removing unnecessary test as the only way to access the edit test is by selecting a run